src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -106,7 +106,10 @@ public function unlock(Request $request)
 
     public function isLocked(Request $request)
     {
-        return is_file($this->getPath($this->getCacheKey($request).'.lck'));
+        $path = $this->getPath($this->getCacheKey($request).'.lck');
+        clearstatcache(true, $path);
+
+        return is_file($path);
```

Comment:
The documentation of [`clearstatcache`](http://php.net/manual/en/function.clearstatcache.php) says `file_exists` is affected.